app/assets/stylesheets/components/_footer.scss

```diff
@@ -23,6 +23,16 @@ body {
     flex-direction: row;
   }
 
+  @include at-media-max('tablet') {
+    .footer__links {
+      flex-wrap: wrap;
+    }
```

Comment:
Can you explain what this style is for? I tested as narrow screen size as iPhone SE (375px) and still wasn't seeing any wrapping, and I wouldn't expect we'd want any.

Reply:
It's maybe a rare case but I tested on Galaxy Fold (simulated) and it came into 280 px wide and caused an awkward break.

without wrap
![Screenshot 2023-09-07 at 8 42 22 AM (2)](https://github.com/18F/identity-idp/assets/135744319/cf706d0f-112a-413f-87ad-93ac2d75e453)
with wrap
![Screenshot 2023-09-07 at 8 42 09 AM (2)](https://github.com/18F/identity-idp/assets/135744319/95e36b79-77ff-4473-ac02-0200bcbbf8f6)

Reply:
Ok, that seems like an extreme case, but seems reasonable that we'd want it to wrap 👍 Thanks for sharing the screenshots.

Aside: It doesn't look great that the logo bumps right up to the edge like that, but maybe we can address that separately.

Reply:
I agree. I've added some X- padding so it doesn't run off the side.

![Screenshot 2023-09-07 at 10 02 35 AM (2)](https://github.com/18F/identity-idp/assets/135744319/b91918d3-e9b1-4e8a-9f1d-f5a8c62a1271)